### ISSUE TITLE
False positive LC0077 for ReadIsolation

### ISSUE THREAD

**Issue by ralf-escher** [StefanMaron/BusinessCentral.LinterCop#1028]
Hi,

the code 
`WarehouseReceiptLine.ReadIsolation(ReadIsolation::UpdLock);`  
shows false positive LC0077 `You must specify open and close parenthesis after 'ReadIsolation'`

additional remark:
the rule does complain about `GetLastErrorText` for missing parenthesis, but not for `GetLastErrorCallStack`.
See #1027

**Comment by jwikman** [comment]
Another variant of above code, that also has a false positive, is when you are using the property access syntax:
```al
Rec.ReadIsolation := ReadIsolation::UpdLock;
```
I assume that it is the `ReadIsolation::` part that is triggering this warning

**Comment by pri-kise** [comment]
There's a similiar false positive for 
```al
currXMLport.TextEncoding := TextEncoding::Windows;
```

**Comment by Arthurvdv** [comment]
I finally found time to resolve these false positives with the Pre-release version [v0.38.1](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.38.1) of the LinterCop.

The false positives are are caused by assigning a value to a method, where the method `[air quotes]` is missing parentheses `[/air quotes]`. This reminded me of a post on direct assigning to a method I've seen on the socials and I've created a new discussion for this [Do not use assignment for function parameters](https://github.com/StefanMaron/BusinessCentral.LinterCop/discussions/1062).

**Comment by jwikman** [comment]
I can confirm that my false positive is gone. 👍
